src/worker/read-wordlist.ts

```diff
@@ -45,3 +45,15 @@ function asNonNegativeInteger(x: unknown): number {
 
   return n;
 }
+
+export async function readManualEntryData(
+  contents: { word: string; count: number | undefined }[]
+): Promise<WordList> {
+  const wordlist: WordList = [];
+  contents.forEach((row) => {
+    const word = row.word;
+    const count = asNonNegativeInteger(row.count || 0);
+    wordlist.push([word, count]);
+  });
+  return wordlist;
```

Comment:
This is a good candidate for `Array.prototype.map()`:

```suggestion
  const wordlist: WordList = contents.map((row) => {
    const word = row.word;
    const count = asNonNegativeInteger(row.count || 0);
    return [word, count];
  });
  return wordlist;
```